Gruntfile.js

```diff
@@ -27,13 +27,25 @@ module.exports = function (grunt) {
             }
         },
         jsonlint: {
-            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json"]
+            all: ["package.json", ".jshintrc", "src/js/*.json", "tests/**/*.json", "demos/**/*.json"]
```

Comment:
I am quite certain I ran into this with the linter previously (it might have been a JSON file in one of Flocking's 3rd party dependencies), but of course now that I check again, everything passes without a problem.

I don't think we should be linting source code for third party projects. However, looking around, it sounds like the appropriate thing to do for `jsonlint` is to have a `.jsonlintignore` file, similar to what we do for `jshint`